### PR TITLE
Use i32 instead of ref

### PR DESCRIPTION
**Pull request by marco-c**
There are still two test failures (tests/gnu/testlet/vm/FieldNotFoundException.java and tests/gnu/testlet/vm/MethodNotFoundException.java).

After this, we should be able to enable garbage collection.

(Depends on #1690)

### CODE REVIEW

File: int.ts [hunk 1]
```diff
@@ -911,48 +911,48 @@ module J2ME {
             sp -= 2;
             continue;
           case Bytecodes.DUP: // ... a -> ... a, a
-            i32[sp] = i32[sp - 1];          ref[sp] = ref[sp - 1];
+            i32[sp] = i32[sp - 1];          i32[sp] = i32[sp - 1];
```
Comment (brendandahl) [inline int.ts]: These need to be removed, not renamed.

Comment (marco-c) [inline int.ts]: I admit I've simply blindly replaced "ref" with "i32", without actually looking at the code. I'll clean the patch up when I fix tests.

File: vm/parser.ts [hunk 2]
```diff
@@ -636,7 +636,7 @@ module J2ME {
               r = classInfo.getMethodByName(name, type);
             }
             if (!r) {
-              throw $.newRuntimeException(classInfo.getClassNameSlow() + "." + fromUTF8(name) + "." + fromUTF8(type) + " not found");
+              throw classInfo.getClassNameSlow() + "." + fromUTF8(name) + "." + fromUTF8(type) + " not found";
```
Comment (brendandahl) [inline vm/parser.ts]: How about we throw a JS object (maybe JavaRuntimeException???) and then in translate exception we'll convert that into a proper java exception. This way we don't have to introduce the try/catch's you added in int.ts.

Comment (marco-c) [inline vm/parser.ts]: Done!

**Pull request merged**